.github/workflows/ci.yml

```diff
@@ -70,7 +70,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: julia-actions/setup-julia@latest
         with:
-          version: '1.5'
+          version: '1.6'
```

Comment:
We should set this to '1', that will automatically use the latest Julia release.